Sources/Moya/Moya+Alamofire.swift

```diff
@@ -68,20 +68,20 @@ public final class CancellableToken: Cancellable, CustomDebugStringConvertible {
 internal typealias RequestableCompletion = (HTTPURLResponse?, URLRequest?, Data?, Swift.Error?) -> Void
 
 internal protocol Requestable {
-    func response(queue: DispatchQueue?, completionHandler: @escaping RequestableCompletion) -> Self
+    func response(callbackQueue: DispatchQueue?, completionHandler: @escaping RequestableCompletion) -> Self
 }
 
 extension DataRequest: Requestable {
-    internal func response(queue: DispatchQueue?, completionHandler: @escaping RequestableCompletion) -> Self {
-        return response(queue: queue) { handler  in
+    internal func response(callbackQueue: DispatchQueue?, completionHandler: @escaping RequestableCompletion) -> Self {
+        return response(queue: callbackQueue) { handler  in
```

Comment:
This argument label remains as `queue` because it's a function from Alamofire :wink: